internal/app/go-filecoin/node/test/setup.go

```diff
@@ -99,6 +99,22 @@ func MustCreateNodesWithBootstrap(ctx context.Context, t *testing.T, additionalN
 	return nodes, cancel
 }
 
+func RunCommandInProcess(ctx context.Context, nd *node.Node, cmd *cmds.Command, optMap cmdkit.OptMap, args ...string) (interface{}, error) {
+	req, err := cmds.NewRequest(ctx, []string{}, optMap, args, nil, cmd)
+	if err != nil {
+		return nil, err
+	}
+	emitter := &testEmitter{}
+	err = cmd.Run(req, emitter, commands.CreateServerEnv(ctx, nd))
+	if err != nil {
+		return nil, err
+	}
+	if emitter.err != nil {
+		return nil, err
+	}
+	return emitter.value, nil
+}
```

Comment:
Interesting, this is an alternative way to run the command. I think all-up I would prefer to stick to the existing way via `RunNodeAPI` (which is still in-process, and maybe you didn't notice). 
- exercises more code, including the CLI de/serialization
- this test emitter doesn't handle mutliple emissions

See protocol_cmd_test. Then un-export the commands.

If there's some reason you need to leave it, please move to a different file from `setup`, maybe 'direct_cmds' or something, so it's more obvious.

Reply:
I removed this and replaced with RunNodeAPI (which I was unaware of).